content/docs/guides/selfbuild/_index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Build from source"
+weight: 2
+description: >
+  Build a development version of TinyGo from source if you want to help improve TinyGo or want to try the latest features.
+---
+
+This page details how to build TinyGo from source. If you would like to install a pre-built binary release, please see our [quick install guide](../../../getting-started/install).
+
+A major dependency of TinyGo is [LLVM](https://llvm.org/). You can either use a version of LLVM already on your system or build LLVM manually. Building manually takes a long time (around an hour depending on how fast your system is) so it is recommended to use a version of LLVM already on your system if that's possible. The links provided below show how to install LLVM one way or the other.
+
+### Repository cloning (before build)
+Start with getting the source code. On Windows, you might want to install the [build dependencies](#build-dependencies) first.
```

Comment:
The `build-dependencies` link does not work for me on the preview.  It might be nice to have a Build Dependencies page with subsections for different operating systems.

Reply:
Hmm, it seems like that's part of the Manual LLVM page now. I'm not sure why it's suggested that the build dependencies should be installed beforehand as cloning the git repo does not really have any effect on what `choco` does. I'm thinking we should probably remove this since build dependencies depend on whether you're manually installing LLVM or if you are using a system installed LLVM